kafka-filter-pypi/entrypoint.py

```diff
@@ -73,9 +73,10 @@ def _exists_in_database(self, entry):
     def consume(self):
         self._init_kafka()
 
+        # todo: separate message consuming logic for lazy ingestion and batch crawl here
```

Comment:
Do you suggest separating the lazy ingestion and the crawling heuristics in different methods rather than including both in the `_extract()` method?

Reply:
We could remove the todo. My first feeling is to use the different logic. After read the `_extract()` method, I decided to make fewer changes and avoid duplications. Thanks for the review.